inc/Remote/ApiCore.php

```diff
@@ -78,7 +78,11 @@ public function getRemoteInfo()
                 'args' => array('string', 'string', 'array'),
                 'return' => 'bool',
                 'doc' => 'Append text to a wiki page.'
-            ), 'dokuwiki.deleteUsers' => array(
+            ), 'dokuwiki.createUser' => array(
+                'args' => array('struct'),
```

Comment:
Why the use of a struct here? Wouldn't it be better to explicitly name the types of arguments we want for user creation?

Reply:
I thought the user was more a struct than a loose collection of parameters. As the api shouldn't change that much I thought it was more future proof.

Reply:
I’m not so experienced with these trade-offs. I wonder if this flexibility is used. There is no event hook that plugins can use, and with these kind of changes more things has to be changed here and remote. So in that case, remote users have to check the version number of the api at all. They haven’t?

Reply:
I'll change that behaviour as proposed by @splitbrain